common/cmake/v2x.cmake

```diff
@@ -94,7 +98,7 @@ function(V2X)
       ${DEPENDS_LIST}
       ${symbiflow-arch-defs_SOURCE_DIR}/utils/vlog/vlog_to_pbtype.py
     COMMAND
-      ${CMAKE_COMMAND} -E env YOSYS=${YOSYS}  ${PYTHON3} ${symbiflow-arch-defs_SOURCE_DIR}/utils/vlog/vlog_to_pbtype.py ${TOP_ARG}
+      ${CMAKE_COMMAND} -E env YOSYS=${YOSYS} PYTHONPATH=${PYTHON_SDF_TIMING_DIR} ${PYTHON3} ${symbiflow-arch-defs_SOURCE_DIR}/utils/vlog/vlog_to_pbtype.py ${TOP_ARG} ${V2X_PB_TYPE_EXTRA_ARGS}
```

Comment:
Please install python-sdf-timing into the conda environment so this isn't needed.

Reply:
Done.